DQM/TrackingMonitorSource/interface/TrackToTrackComparisonHists.h

```diff
@@ -0,0 +1,166 @@
+  generalME referenceTracksMEs_;
+  generalME matchedReferenceTracksMEs_;
+
+  // monitored tracks All and unmatched
+  generalME monitoredTracksMEs_;
+  generalME unMatchedMonitoredTracksMEs_;
+
+  // Track matching statistics
+  matchingME matchTracksMEs_;
+
+  double Eta_rangeMin, Eta_rangeMax;
+  unsigned int Eta_nbin;
+  double Pt_rangeMin, Pt_rangeMax;
+  unsigned int Pt_nbin;  //bool useInvPt;   bool useLogPt;
+  double Phi_rangeMin, Phi_rangeMax;
+  unsigned int Phi_nbin;
+  double Dxy_rangeMin, Dxy_rangeMax;
+  unsigned int Dxy_nbin;
+  double Dz_rangeMin, Dz_rangeMax;
+  unsigned int Dz_nbin;
+
+  double ptRes_rangeMin, ptRes_rangeMax;
+  unsigned int ptRes_nbin;
+  double phiRes_rangeMin, phiRes_rangeMax;
+  unsigned int phiRes_nbin;
+  double etaRes_rangeMin, etaRes_rangeMax;
+  unsigned int etaRes_nbin;
+  double dxyRes_rangeMin, dxyRes_rangeMax;
+  unsigned int dxyRes_nbin;
+  double dzRes_rangeMin, dzRes_rangeMax;
+  unsigned int dzRes_nbin;
+
+  //std::vector<int> totSIMeta,totRECeta,totASSeta,totASS2eta,totloopeta,totmisideta,totASS2etaSig;
+  //std::vector<int> totSIMpT,totRECpT,totASSpT,totASS2pT,totlooppT,totmisidpT;
+  //std::vector<int> totSIM_hit,totREC_hit,totASS_hit,totASS2_hit,totloop_hit,totmisid_hit;
+  //std::vector<int> totSIM_phi,totREC_phi,totASS_phi,totASS2_phi,totloop_phi,totmisid_phi;
+  //std::vector<int> totSIM_dxy,totREC_dxy,totASS_dxy,totASS2_dxy,totloop_dxy,totmisid_dxy;
+  //std::vector<int> totSIM_dz,totREC_dz,totASS_dz,totASS2_dz,totloop_dz,totmisid_dz;
+
+  //std::vector<int> totSIM_vertpos,totASS_vertpos,totSIM_zpos,totASS_zpos;
+  //std::vector<int> totSIM_vertcount_entire,totASS_vertcount_entire,totREC_vertcount_entire,totASS2_vertcount_entire,totASS2_vertcount_entire_signal;
+  //std::vector<int> totSIM_vertcount_barrel,totASS_vertcount_barrel,totREC_vertcount_barrel,totASS2_vertcount_barrel;
+  //std::vector<int> totSIM_vertcount_fwdpos,totASS_vertcount_fwdpos,totREC_vertcount_fwdpos,totASS2_vertcount_fwdpos;
+  //std::vector<int> totSIM_vertcount_fwdneg,totASS_vertcount_fwdneg,totREC_vertcount_fwdneg,totASS2_vertcount_fwdneg;
+  //std::vector<int> totSIM_vertz_entire,totASS_vertz_entire;
+  //std::vector<int> totSIM_vertz_barrel,totASS_vertz_barrel;
+  //std::vector<int> totSIM_vertz_fwdpos,totASS_vertz_fwdpos;
+  //std::vector<int> totSIM_vertz_fwdneg,totASS_vertz_fwdneg;
+  //std::vector<int> totREC_algo;
+  //std::vector<int> totREC_ootpu_entire, totASS2_ootpu_entire;
+  //std::vector<int> totREC_ootpu_barrel, totASS2_ootpu_barrel;
+  //std::vector<int> totREC_ootpu_fwdpos, totASS2_ootpu_fwdpos;
+  //std::vector<int> totREC_ootpu_fwdneg, totASS2_ootpu_fwdneg;
+  //std::vector<int> totREC_ootpu_eta_entire, totASS2_ootpu_eta_entire;
+  //std::vector<int> totASS2_itpu_eta_entire, totASS2_itpu_eta_entire_signal, totASS2_itpu_vertcount_entire, totASS2_itpu_vertcount_entire_signal;
+  //std::vector<int> totFOMT_eta, totFOMT_vertcount;
+  //std::vector<int> totCONeta, totCONvertcount, totCONzpos;
```

Comment:
Could you please remove all this commented code? You can always access it by the commit history, Thanks

Reply:
Done